contents/tutorials/android-feature-flags.md

```diff
@@ -0,0 +1,235 @@
+---
+title: "How to set up feature flags in Android"
```

Comment:
this page is very similar to https://posthog.com/tutorials/android-ab-tests
what is the main difference? should we kill one of them?

Reply:
THis is intentional. They're targeting different keywords for google search. One is for "android ab tests" and the other for "android feature flags" :)